src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -160,5 +163,10 @@ class PdlClient(
         private val FOLKEREG_IDENTER_CACHE_KEY_PREFIX = "pdl-folkereg-identer"
         private val CACHE_EXPIRE_SECONDS = 24L * 3600
         private val logger = LoggerFactory.getLogger(PdlClient::class.java)
+
+        // Se behandlingskatalog https://behandlingskatalog.intern.nav.no/
+        // Behandling: Sykefraværsoppfølging: Vurdere behov for oppfølging og rett til sykepenger etter §§ 8-4 og 8-8
+        private const val BEHANDLINGSNUMMER_HEADER_KEY = "behandlingsnummer"
+        private const val BEHANDLINGSNUMMER_HEADER_VALUE = "B426"
```

Comment:
Var det ikke snakk om at vi hadde en egen behandling for dialogmøter? Eller skal vi bare bruke en generell syfo-behandling på alle kall?

Reply:
Hmm, godt spørsmål. Hvor viktig er det at behandlingsnummeret er nøyaktig🤔 Virker safe å bare bruke behandling for dialogmøter her så kan godt gjøre det 😊 Bra sett!

Reply:
Vet ikke hvor viktig det er, men virker riktigst å bruke den mest spesifikke :+1: Ser vi har en egen for [behandlerdialog](https://behandlingskatalog.intern.nav.no/process/system/MODIA_SYKEFRAVAER/e4811479-b454-4a75-8923-6cbc4da8ee16) og [oppfølgingstilfelle](https://behandlingskatalog.intern.nav.no/process/system/MODIA_SYKEFRAVAER/50f0223f-ab56-40ed-85ff-8e6a7b2b051c) også.